File: Python/Exercice3_1.py
#coding: utf-8
import sys

if len(sys.argv) == 1:
    print("Vous n'avez entrer aucun arguments")
    sys.exit(0)
else:
    print(f"Vous avez entre {len(sys.argv)-1} arguments")
    a = int(sys.argv[1])
    b = int(sys.argv[2])
    print(f"L'addition des chiffres que vous avez entre est: {a} + {b} = {a+b}")

File: C/Exercice3_1.c
#include <stdlib.h>
#include <stdio.h>

int main(int argc, char* argv[])
{
    int a, b;
    if(argc == 1)
    {
        printf("Vous n'avez entrer aucun arguments\n");
        exit(0);
    }
    else if(argc >= 2)
    {
        printf("Vous avez entre %d arguments\n", argc);
        a = atoi(argv[1]);
        b = atoi(argv[2]);
        printf("L'addition des chiffres que vous avez entre est: %d + %d = %d\n", a, b, a+b);
    }
    return 0;
}